queue_services/entity-emailer/src/entity_emailer/email_templates/common/16px.html

```diff
@@ -0,0 +1,3 @@
+<!-- empty paragraph for spacing -->
+<!-- height = 0 + 16 = 16px -->
+<p style="margin-top: 16px !important;"></p>
```

Comment:
This is truly an "empty" paragraph, so the `p` tag itself doesn't add any height. Hence the height is truly 16px here.